third_party/update-docusaurus-search-local.sh

```diff
@@ -2,15 +2,15 @@
 
 set -e
 
-COMMIT_HASH="5c51edc1d46b2739236cd16b67e3f6e2ae7bd0cb"
+COMMIT_HASH="9093d71039f35409535ca77fdf2198d1d87291bf"
 
 cd third_party
 
 TMP_ZIP_FILE="docusaurus-search-local.zip"
 
 rm -rf docusaurus-search-local/
 
-curl -o "${TMP_ZIP_FILE}" "https://codeload.github.com/easyops-cn/docusaurus-search-local/zip/${COMMIT_HASH}"
+curl --fail -o "${TMP_ZIP_FILE}" "https://codeload.github.com/easyops-cn/docusaurus-search-local/zip/${COMMIT_HASH}"
```

Comment:
what is going on here?

Reply:
It will return an error on server errors where the HTTP response code is 400 or greater)